GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -197,7 +202,8 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            EditorGUILayout.TextField("c:\\base.apk", GUILayout.MinWidth(FieldMinWidth));
+            QuickDeployConfig.Config.apkFileName = EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName,
```

Comment:
Let's think about the semantics of when we decide to save the configuration. If we want to save whenever these fields are modified, it may be better to use a setter here that directly chains the call to save the configuration.

Reply:
Discussed offline. Unsure of save model, so resolving.